File: Lab_1.pas
program test (input, output);
var
  S, i, N: integer;
begin
  write('Please enter value for N:' );
  readln(N);
  S := 0;
  for i := 0 to N - 1 do 
  begin
    if i mod 2 = 0 then
    begin
      S := S + i ;
      writeln(' Result is ', i, ' S=', S);
    end; 
  end;
  writeln('Final result is S=', S);
end.

File: Lab_1.py
n = int();
s = int();
i = int();
print('Please enter value for N\n');
n = int(input());
s = 0;
for i in range(1, n, 1):
    if i % 2 == 0:
        s = s + i;
        print("\nCurrent value i=", i, " S= ", s);
print("Final result suma is S=", s);
